### PR TITLE
Cleanup/remove deprecated metrics

### PR DESCRIPTION
**Pull request by asdacap**
- Remove deprecated metrics.

#### What types of changes does your code introduce?

- [X] Refactoring

#### Requires testing

- [X] Yes
- [ ] No

#### If yes, did you write tests?

- [ ] Yes
- [X] No

#### Notes on testing

- Seems to work

### CODE REVIEW

**Review by LukaszRozmej — APPROVED** [review]

File: src/Nethermind/Nethermind.TxPool/TxPool.cs [hunk 1]
```diff
@@ -804,8 +804,8 @@ private static void WriteTxPoolReport(in ILogger logger)
 * Blobs:                {Metrics.BlobsInBlock,24:N0}
 ------------------------------------------------
 Db usage:
-* BlobDb writes:        {Db.Metrics.BlobTransactionsDbWrites,24:N0}
-* BlobDb reads:         {Db.Metrics.BlobTransactionsDbReads,24:N0}
+* BlobDb writes:        {Db.Metrics.DbWrites.GetValueOrDefault("BlockTransactions"),24:N0}
+* BlobDb reads:         {Db.Metrics.DbReads.GetValueOrDefault("BlockTransactions"),24:N0}
```
Comment (LukaszRozmej) [inline src/Nethermind/Nethermind.TxPool/TxPool.cs:808]: Blob not Block

File: src/Nethermind/Nethermind.Network/P2P/Analyzers/MetricsDisconnectsAnalyzer.cs [hunk 2]
```diff
@@ -10,102 +10,14 @@ public class MetricsDisconnectsAnalyzer : IDisconnectsAnalyzer
     {
         public void ReportDisconnect(DisconnectReason reason, DisconnectType type, string details)
         {
-            EthDisconnectReason ethReason = reason.ToEthDisconnectReason();
```
Comment (LukaszRozmej) [inline src/Nethermind/Nethermind.Network/P2P/Analyzers/MetricsDisconnectsAnalyzer.cs:13]: Whole method could now be:

```
(type == DisconnectType.Remote ? Metrics.RemoteDisconnectsTotal : Metrics.LocalDisconnectsTotal).Increment(reason);
```

File: src/Nethermind/Nethermind.TxPool/TxPool.cs (hunk 1 above)
Comment (asdacap) [reply src/Nethermind/Nethermind.TxPool/TxPool.cs]: Done

File: src/Nethermind/Nethermind.Network/P2P/Analyzers/MetricsDisconnectsAnalyzer.cs (hunk 2 above)
Comment (asdacap) [reply src/Nethermind/Nethermind.Network/P2P/Analyzers/MetricsDisconnectsAnalyzer.cs:13]: Done

**Pull request merged**